### ISSUE TITLE
Updating the example in core-is-installed

### ISSUE THREAD

**Issue by ntchambers** [wp-cli/core-command#144]
In the example for [wp core is-installed](https://developer.wordpress.org/cli/commands/core/is-installed/), it shows the following example:

```
# Bash script for checking whether WordPress is installed or not
if ! $(wp core is-installed); then
    wp core install
fi
```

which looks to still be in the latest [source](https://github.com/wp-cli/core-command/blob/master/src/Core_Command.php#L350-L353). In Bash, `if` takes and executes a command, so it should ideally just be:

```
# Bash script for checking whether WordPress is installed or not
if ! wp core is-installed; then
    wp core install
fi
```

At the moment it still works since `is-installed` doesn't output anything, but  if it ever did, it will try to execute the output as a command (and use that as an exit status):

```
$ somecmd() { echo this is not a command; return 0; }
$ if ! $(somecmd); then echo command failed; fi
bash: this: command not found
command failed
$ if ! somecmd; then echo command failed; fi
this is not a command
```

Removing the `$()` should allow it to operate as expected, regardless of how `is-installed` works.

**Comment by ntchambers** [comment]
It also [gets flagged in shellcheck](https://github.com/koalaman/shellcheck/wiki/SC2091) for a few reasons.

**Comment by schlessera** [comment]
Good catch! Are you interested in creating a PR against https://github.com/wp-cli/core-command/blob/master/src/Core_Command.php#L350-L353 to fix this?

**Comment by ntchambers** [comment]
Yeah, definitely!

**Comment by schlessera** [comment]
Fixed via #145
